rosys/hardware/robot_brain.py

```diff
@@ -25,6 +25,7 @@ class RobotBrain:
     def __init__(self, communication: Communication) -> None:
         self.LINE_RECEIVED = Event()
         """a line has been received from the microcontroller (argument: line as string)"""
+        self.FLASH_P0_COMPLETE = Event()
```

Comment:
Let's add a descriptive docstring like for all the other RoSys events. An event name tends to be a bit short for describing when _exactly_ the event is fired.